src/Diagnostics.DataProviders/DataProviders/GeoMasterDataProvider.cs

```diff
@@ -200,12 +201,11 @@ public string RemovePIIFromSettings(string content)
         /// }
         /// </code>
         /// </example>
-        public async Task<VnetValidationRespone> VerifyHostingEnvironmentVnet(string subscriptionId, string vnetResourceGroup, string vnetName, string vnetSubnetName, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
Why do we want to remove all these?

Reply:
It's all about avoiding potential deadlocking.

https://stackoverflow.com/questions/21033150/any-difference-between-await-task-run-return-and-return-task-run

With those changes, you will not be able to set a breakpoint at the await or return statements, but you could still set it at their callers.